aspnetcore/security/anti-request-forgery/sample/AngularSample/Startup.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.IO;
+using Microsoft.AspNetCore.Antiforgery;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace AngularSample
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Angular's default header name for sending the XSRF token.
+            services.AddAntiforgery(options => options.HeaderName = "X-XSRF-TOKEN");
+
+            services.AddSingleton<TodoRepository>();
+        }
+
+public void Configure(IApplicationBuilder app, 
+    IAntiforgery antiforgery, 
+    TodoRepository repository)
+{
+    app.Use(next => context =>
+    {
+        if (
+            string.Equals(context.Request.Path.Value, "/", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
Pop a browser window and make the widest window that doesn't have a TOC or the left or the right. In general that give you the max width of code - that is any code that creates a horizontal scroll bar needs to be shortened.

In this case you need a newline after ``"/"